pkg/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+type Option func(*ClientConfig)
+
+// ClientConfig defines external client config
+type ClientConfig struct {
+	Account   string
+	Namespace string
+	Prefix    string
+	Dims      map[string]string
+}
+
+// Interface represents metrics interface
+type Interface interface {
+	Clone(opts ...Option) Interface
+	Send([]byte) (int, error)
+	EmitFloat(string, float64) error
+	EmitFloatWithDims(string, float64, ...string) error
+	EmitGauge(string, int64) error
+	EmitGaugeWithDims(string, int64, ...string) error
+}
```

Comment:
I think this interface can be pared right back to `Emit{Float,Gauge}(metric string, value whatever, dims map[string]string) error - then you sub in namespace, accounts, ts, marshal lock and write.

I don't think that Option, Prefix or Dimensions are needed.

Without the prefix logic, code that just says Emit("frontend.duration") it is simpler, and it is more greppable.